Arduino/Ground Systems/Controls/controls.ino

```diff
@@ -44,7 +44,7 @@ void loop()
       {
```

Comment:
Also, can't seem to make a comment on the specific line, but the same removal should also apply to the global `pins` array init on Line 6: https://github.com/UCLA-Rocket-Project/AresSoftware/blob/main/Arduino/Ground%20Systems/Controls/controls.ino#L6

Reply:
Ah, makes sense! We'll leave it up to him, then, to remove that!